index.html

```diff
@@ -0,0 +1,240 @@
+</head>
+<body>
+    <!--
+        1. Follow the instructions atop the [Recipes project](/courses/foundations/lessons/recipes) to set 
+        up a GitHub repository for this project (of course you'll need to change the title). 
+        2. Create a blank HTML document. 
+        3. Come up with a plan -- which sections of the page should be grouped together
+         (e.g. the navigation bar vs the video vs the sidebar etc)? 
+        4. Create empty semantic HTML elements to enclose those sections.
+        5. Use basic CSS to size and position them appropriately on the page. 
+          It may be helpful to make their backgrounds different colors so you can see the different sections.
+           Don't forget to use your browser's developer tools (right click on the page, click "inspect element")!
+        6. Now let's start filling in those divs with content by creating the navbar. 
+        7. Watch [this YouTube video on embedding a YouTube video in a webpage](https://www.youtube.com/watch?v=lJIrF4YjHfQ&feature=emb_title).
+        8. Embed a YouTube video of your choice into your page.
+        9. Next, build the sidebar section where previews of suggested videos show up. 
+           Obviously don't worry about generating those previews... just grab some thumbnail-sized images and plug them in.
+        10. Next, finish up by filling in some of the elements below the video -- the title, description and number of views.
+             Ignore the icons (many of them are produced using sprites or more advanced techniques 
+             so you won't be able to just grab them by copying them). 
+        11. Push your solution up to GitHub.
+    --> 
+ <div class="cont">
+    <div id="header">
+         <form>
+            <img src="img/youtube_logo.png" class="youtube" >
+            <input type="text" name="input" placeholder="Search" class="search" >
+            <button type="submit" class="btn">
+                <img src="img/search.png"  >
+            </button>
+         </form>
+       
+        </div>
+       
+        <div class="main"> 
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/Pwzd81hHffY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen>
+             </iframe>  
+             <ul>
+                 
+                 <li  class="frameLi"><img src="img/name.png" class="subscribtion"></li>
+                 <li class="frameLi"  >&nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;<img src="img/like.png" class="reaction">154k</li>
+                 <li   class="frameLi">&nbsp; &nbsp;<img src="img/dont-like.png" class="reaction"> </li>
+                 <li  class="frameLi" >&nbsp; &nbsp; &nbsp;<img src="img/download.png"class="reaction" >Download</li>
+                 <li  class="frameLi" >&nbsp; &nbsp; &nbsp;<img src="img/option.png" class="reaction"></li>
+             </ul> 
```

Comment:
Great job finding a better approach to aligning the item 👍 .  Kindly tell the student why the inline and &nbsp should not be used. This will make the student know why they are making changes 😉